metaflow/plugins/env_escape/configurations/emulate_test_lib/overrides.py

```diff
@@ -0,0 +1,90 @@
+from metaflow.plugins.env_escape.override_decorators import (
```

Comment:
can we move `emulate_test_lib` to `/tests`?

Reply:
Unfortunately not; the full escape hatch mechanism reads things from configurations/ and looks for things called emulate_* so it needs to be here.